pkg/loki/flow_query.go

```diff
@@ -180,6 +180,18 @@ func (q *FlowQueryBuilder) addLineFilters(key string, values []string, not bool,
 	if len(values) == 0 {
 		return
 	}
+
+	var isArray bool
+	if q.config.Deduper.Merge {
+		switch key {
+		case "FlowDirection", "Interface":
+			key = fmt.Sprintf("%ss", key)
+			isArray = true
+		default:
+			isArray = false
+		}
+	}
```

Comment:
Yes !

Currently, when deduper merged is enabled, we have both single and plural fields as:
FlowDirection, Interface, FlowDirections, Interfaces

I'm forcing a `lineMatch`` on plural fields here to avoid any query running on single fields.
In the future, I expect fully removing FlowDirection and Interface fields.